Buildings/Controls/OBC/ASHRAE/G36_PR1/AHUs/MultiZone/SetPoints/VAVSupplyFan.mo

```diff
@@ -391,16 +394,16 @@ parameters as a starting point:
 <table summary=\"summary\" border=\"1\">
 <tr><th> Variable </th> <th> Value </th> <th> Definition </th> </tr>
 <tr><td>Device</td><td>AHU Supply Fan</td> <td>Associated device</td></tr>
-<tr><td>SP0</td><td>120 Pa (0.5 inches)</td><td>Initial setpoint</td></tr>
-<tr><td>SPmin</td><td>25 Pa (0.1 inches)</td><td>Minimum setpoint</td></tr>
+<tr><td>SP0</td><td>iniSet</td><td>Initial setpoint</td></tr>
```

Comment:
Wrap these codes into a `<code>...</code>` (then the CDL exporter will automatically add a hyperlink)